snap/implicit_test.go

```diff
@@ -71,3 +73,174 @@ func (s *InfoSnapYamlTestSuite) TestImplicitSlotsAreRealInterfaces(c *C) {
 		c.Check(known[ifaceName], Equals, true)
 	}
 }
+
+func (s *InfoSnapYamlTestSuite) TestAddImplicitHooksNoHooks(c *C) {
+	dirs.SetRootDir(c.MkDir())
+	defer func() { dirs.SetRootDir("") }()
+
+	yaml := `name: foo
+version: 1.0`
+	info := snaptest.MockSnap(c, yaml, &snap.SideInfo{Revision: snap.R(42)})
+
+	// Now load implicit hooks
+	snap.AddImplicitHooks(info)
```

Comment:
@kyrofa I think it would be better  (the interfaces implicit stuff is actually exported) but these are actually testing internals helpers whose existence is just a detail